app/models/ncr/work_order.rb

```diff
@@ -6,15 +6,17 @@ def self.table_name_prefix
 
   DATA = YAML.load_file("#{Rails.root}/config/data/ncr.yaml")
 
+  EXPENSE_TYPES = %w(BA61 BA80)
+
+  BUILDING_NUMBERS = DATA['BUILDING_NUMBERS']
+  OFFICES = DATA['OFFICES']
+
   class WorkOrder < ActiveRecord::Base
     # In practice, each work order only has one proposal
     has_many :proposals, as: :client_data
```

Comment:
The reliance on cart is going away, right? Making it painful to connect a work order to a cart is a way to speed up that process.